src/Constants/collection.js

```diff
@@ -0,0 +1,19 @@
+const collection = [
```

Comment:
where are you using this collection?

Reply:
I thought of it before pushing and it must be used when creating a card, like when picking the type, lets say memories, the corresponding color is used. what do you think?